src/java/org/apache/cassandra/config/Config.java

```diff
@@ -257,11 +257,14 @@
     public volatile long native_transport_max_concurrent_connections_per_ip = -1L;
     public boolean native_transport_flush_in_batches_legacy = false;
     public volatile boolean native_transport_allow_older_protocols = true;
-    public volatile long native_transport_max_concurrent_requests_in_bytes_per_ip = -1L;
-    public volatile long native_transport_max_concurrent_requests_in_bytes = -1L;
+    @Replaces(oldName = "native_transport_max_concurrent_requests_in_bytes_per_ip", converter = Converters.BYTES_CUSTOM_DATASTORAGE, deprecated = true)
+    public volatile DataStorageSpec native_transport_max_concurrent_requests_per_ip = new DataStorageSpec("0B");
+    @Replaces(oldName = "native_transport_max_concurrent_requests_in_bytes", converter = Converters.BYTES_CUSTOM_DATASTORAGE, deprecated = true)
+    public volatile DataStorageSpec native_transport_max_concurrent_requests = new DataStorageSpec("0B");
```

Comment:
So we now have `native_transport_max_concurrent_requests`, `native_transport_max_concurrent_requests_per_ip`, and `native_transport_max_requests_per_second`, but only the third one actually deals with requests and not the amount of data in flight. The nice thing about, for instance `native_transport_max_concurrent_requests_in_bytes` was that it made that a bit clearer. Looking at the internals, we use phrases like "bytes in flight" to describe this. What about something like...

`native_transport_max_concurrent_requests_in_bytes` -> `native_transport_max_request_data_in_flight`
...and...
`native_transport_max_concurrent_requests_in_bytes_per_ip ` -> `native_transport_max_request_data_in_flight_per_ip`
?

Reply:
Was concurrent removed intentionally?

Reply:
I would go so far as to change the names in DD as well.

Reply:
Sure, I was wondering whether you removed "_concurrent_" from the parameters names intentionally?

Reply:
I did. "Concurrent" feels redundant when we also say "in flight" :/

Reply:
As per our conversation, I used the suggested names and fixed the setters and the respective Converter that -1 should mean null and we set default value as per the DD. I will open separate ticket for 4.0 to fix the setter -1 to point to the default value.